src/scanner.c

```diff
@@ -1144,15 +1115,60 @@ void GetSymbol ( void )
   }
 }
 
-Obj FuncALL_KEYWORDS(Obj self) {
-    Obj l;
+static const char * AllKeywords[] = {
+    "and",
+    "atomic",
+    "break",
+    "continue",
+    "do",
+    "elif",
+    "else",
+    "end",
+    "false",
+    "fi",
+    "for",
+    "function",
+    "if",
+    "in",
+    "local",
+    "mod",
+    "not",
+    "od",
+    "or",
+    "readonly",
+    "readwrite",
+    "rec",
+    "repeat",
+    "return",
+    "then",
+    "true",
+    "until",
+    "while",
+    "quit",
+    "QUIT",
+    "IsBound",
+    "Unbind",
+    "TryNextMethod",
+    "Info",
+    "Assert",
+};
 
```

Comment:
I'd probably just have let clang-format format this more compactly, but this is fine.

Reply:
I can do that, and it is very compact then, *but* it means that if anything changes, then the resulting diff (if one immediately applies `clang-format` again gets a lot bigger. This is why I generally prefer formatting lists, enums etc. with one entry per line: to keep diffs understandable.

But I guess we are not going to change the list of keywords that often *shrug*